crates/liboci-cli/README.md

```diff
@@ -24,3 +25,6 @@ Interface](https://github.com/opencontainers/runtime-tools/blob/master/docs/comm
 | run        | ✅         |                   | ✅   | ✅   | ✅    |
 | spec       | ✅         |                   | ✅   | ✅   | ✅    |
 | update     |            |                   | ✅   | ✅   |       |
+
+🏴 The `features` subcommand is [implemented](https://github.com/opencontainers/runc/pull/3296)
+in `runc`, but not documented.
```

Comment:
This is defined by OCI Runtime Spec.
https://github.com/opencontainers/runtime-spec/blob/07dd1f41412c2c0b8f8d535c8aeaa9ced8dc3f5d/features-linux.md

Reply:
Thanks for the link, @utam0k. This is probably because [when I wrote the code (March 9)](https://github.com/containers/youki/commit/947b1ee1c0a82e379997a29983c80e9ee1666245) the documentation was [not committed yet (March 21)](https://github.com/opencontainers/runtime-spec/commit/689874fc76bf43d5a79cb733bcc7bcf7756bdb0e). I will fix it.

Reply:
@utam0k Implemented in commit 4381887